src/muya/lib/index.js

```diff
@@ -74,6 +74,11 @@ class Muya {
             if (hasTable) {
               console.warn('There was a problem with the table deletion.')
             }
+            const lineRemovedUnExpected = Array.from(removedNodes).some(node => node.nodeType === 1 && node.classList.contains('ag-paragraph-content'))
```

Comment:
I think this should be named `lineRemovedUnexpected` because `un` is just a prefix and please add a comment and/or link to the associated GitHub issue of better understanding.